client/components/ui/data-table/components/cells/editable-cell.tsx

```
@@ -0,0 +1,150 @@
+            onCancel();
+        } else if (e.key === "Tab") {
+            e.preventDefault();
+            e.stopPropagation();
+            await handleSave();
+            e.shiftKey ? onFocusPrev?.() : onFocusNext?.();
+        }
+    };
+
+    // Render: Delegate to column's custom render function
+    return (
+        <div className="relative w-full" onKeyDown={handleKeyDown}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)